apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -467,6 +467,113 @@ expression: "&schema"
           },
           "additionalProperties": false
         },
+        "supergraph": {
+          "description": "The supergraph stage request/response configuration",
+          "default": {
+            "request": {
+              "headers": false,
+              "context": false,
+              "body": false,
+              "sdl": false,
+              "path": false,
+              "method": false
```

Comment:
yes, I think there should be other fields. I didn't know if we wanted to have an entire `response` field or be able to split the response parts when sending to the coprocessor?

Reply:
It's hard to know/decide what's required at the different stages... I tend to err on the side of "if you think someone might want it, and it's easy to provide, then include it, since it can be configured out".